builtin/providers/cloudstack/resource_cloudstack_loadbalancer_rule.go

```diff
@@ -138,7 +145,10 @@ func resourceCloudStackLoadBalancerRuleRead(d *schema.ResourceData, meta interfa
 	cs := meta.(*cloudstack.CloudStackClient)
 
 	// Get the load balancer details
-	lb, count, err := cs.LoadBalancer.GetLoadBalancerRuleByID(d.Id())
+	lb, count, err := cs.LoadBalancer.GetLoadBalancerRuleByID(
+		d.Id(),
+		cloudstack.WithProject(d.Get("project").(string)),
```

Comment:
That will never happen... `d.Get()` will always give you a zero value if you didn't set it in your config.